USAGE.md

```diff
@@ -10,6 +10,7 @@
 - [Faker.Beer](lib/faker/beer.ex)
 - [Faker.Beer.En](lib/faker/beer/en.ex)
 - [Faker.Blockchain.Bitcoin](lib/faker/bitcoin.ex)
+- [Faker.BlockChain.CryptoCoin](lib/faker/blockchain/crypto_coin.ex)
```

Comment:
It's `Faker.Blockchain`.

Reply:
@thadeubrito  could you make this change?